angelssmelting/prototypes/technology/smelting-cobalt.lua

```diff
@@ -76,7 +77,8 @@ data:extend(
         ingredients = {
           {type = "item", name = "automation-science-pack", amount = 1},
           {type = "item", name = "logistic-science-pack", amount = 1},
-          {type = "item", name = "chemical-science-pack", amount = 1}
+          {type = "item", name = "chemical-science-pack", amount = 1},
+          {type = "item", name = "production-science-pack", amount = 1}
         },
         time = 30
       },
```

Comment:
This also means that `angels-cobalt-casting-2` should be purple science and depend on metallurgy-4 and `angels-cobalt-casting-3`+`"angels-cobalt-smelting-3"` should be purple+yellow science and depend on metallurgy-5

The changes for industry science mode should be automagically processed by the replacement script.

Similar changes are required for gold.

Reply:
Thanks, i knew that i was missing some of the logic around the smelting tech tree, hopefully this is now ready to roll